cms/admin.py

```diff
@@ -26,3 +27,35 @@ class Media:
 @admin.register(Menu)
 class MenuAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(BlogPage)
+class BlogPageAdmin(admin.ModelAdmin):
+    list_display = (
+        "title",
+        "published",
+        "slug",
+        "created",
+    )
+    list_display_links = ("title",)
+    list_editable = ("published",)
+    fields = (
+        "published",
+        "title",
+        "title_image",
+        "slug",
+        "template",
```

Comment:
Do we really need a template field, since the content is now written directly in the card content?

Reply:
no template selection needed